templates/user/settings/profile.tmpl

```diff
@@ -40,6 +40,20 @@
 					<input id="location" name="location"  value="{{.SignedUser.Location}}">
 				</div>
 
+					<div class="field">
+						<label for="language">{{.i18n.Tr "settings.language"}}</label>
+						<div class="ui language selection dropdown" id="language">
+							<input name="language" type="hidden">
+							<i class="dropdown icon"></i>
+							<div class="text">{{.LangName}}</div>
+							<div class="menu">
+							{{range .AllLangs}}
+								<div class="item{{if eq $.Lang .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
```

Comment:
You should check if `.Lang` is equal with user's language in database and not with the current language. 
If I set `german` as my language but change it manually by selecting another language (example: `english`) from the dropdown in the footer, the selected language in the form will be `english`. The expected selected language should be `german`.

Reply:
Fixed that.